src/styles/card_apple/style.scss

```diff
@@ -0,0 +1,120 @@
+@import url(./styleStars.css);
+
+* {
+  margin: 0;
+  text-decoration: none;
+}
```

Comment:
file should contain only that styles which related to element, 
also, it's bad practice to set styles in `*` tag